.gitignore

```diff
@@ -21,9 +21,10 @@ src/*-clone
 src/buildpacks/*-clone
 /*.tf
 /*.tf.json
-
+bin/settings/kube/ca.env
 # `make dist` output of various forms
 output/
 /kube/bosh/
 /kube/bosh-task/
+/kube/secrets/
```

Comment:
Should we just ignore `kube/` instead of the 3 lines above? I don't see that we want to check anything under that directory into git.